### PR TITLE
style: change and apply spotless with indentation 4 spaces

### LINKED ISSUE #63: change and indentation for spotless to 4
**Describe the chore**
indentation for spotless to 2 spaces. To change and apply spotless with indentation 4 spaces.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by immadisairaj**
Fixed #63

Changes: 
Changed the spotless style in gradle
applied spotless
removed unnecessary typecasting
made xml files in proper format

### CODE REVIEW

File: app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java [hunk 1]
```diff
@@ -19,100 +19,100 @@
...
+
+        radioGroup.setOnCheckedChangeListener(
+                new RadioGroup.OnCheckedChangeListener() {
+                    public void onCheckedChanged(RadioGroup group, int checkedId) {
+                        switch (checkedId) {
+                            case R.id.radio_donate:
+                                totalLayout.setVisibility(View.VISIBLE);
+                                flag = 0;
+                                Toast.makeText(
+                                                getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_LONG)
```
Comment (yashk2000) [inline app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: This seems like too much indentation. I think 2 space indentation is good enough.

Comment (immadisairaj) [reply app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: Those has to be in same line. I guess that is the reason it is giving that big indentation

Comment (immadisairaj) [reply app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: Those has to be in same line. I guess that is the reason it is giving that big indentation

Comment (yashk2000) [reply app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: Yeah, I saw other changes too. I thought that 4 tabs are too much. According to me, 2 are enough.

File: app/src/main/java/org/amfoss/templeapp/activities/InsertData.java [hunk 2]
```diff
@@ -21,147 +21,147 @@
...
+        nameET = (EditText) findViewById(R.id.name);
+        paid = (CheckBox) findViewById(R.id.paid_check);
+        poojaType = (Spinner) findViewById(R.id.spinner1);
+        totalLayout = (LinearLayout) findViewById(R.id.total_View);
+        moneyDonated = (EditText) findViewById(R.id.money_donated);
+
+        radioGroup = (RadioGroup) findViewById(R.id.radiogroup);
+
+        radioGroup.setOnCheckedChangeListener(
+                new RadioGroup.OnCheckedChangeListener() {
```
Comment (yashk2000) [inline app/src/main/java/org/amfoss/templeapp/activities/InsertData.java]: For instance here. 4 tabs are too much. 2 are enough.

File: app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java [hunk 3]
```diff
@@ -19,100 +19,100 @@
...
-  String id;
-  String name;
-  RadioGroup radioGroup;
-  RelativeLayout totalLayout;
-  int flag;
-  private Button delete;
-  private EditText uid1ET;
-
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
```
Comment (immadisairaj) [inline app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java]: @yashk2000 suppose if you see here 2 space indentation doesn't look good

**Comment by yashk2000** [comment]
@immadisairaj yes. I viewed it in andorid studio. There is looks better :+1:

**Pull request merged**